### DISCUSSION TITLE
Executing metrics update for project

### DISCUSSION THREAD

**Discussion by savujevi** [DependencyTrack/dependency-track#1379] [Q&A] [answered]
Hello,

what does "Executing metrics update for project" mean ?

Does it mean that if the vulnerability databases are update during the time, the different projects and their boms are checked again against new findings ?

Or does DT doesn't do this at all ?

Thank you for your help.

Best regards,
   Sascha Vujevic

**Comment by stevespringett** [comment, marked as answer]
Project metrics calculate the number of vulnerabilities by criticality, number audited, policy violations, etc.

Refer to https://github.com/DependencyTrack/dependency-track/blob/master/src/main/java/org/dependencytrack/model/ProjectMetrics.java for the fields that are captured in project metrics.

**Reply by savujevi** [reply]
Thanks.

Does new vulnerabilities are shown on an existing project after a new import from e.g. Sonatype OSS Index was made or do I have to reimport the bomfile ?

Thanks.

**Reply by stevespringett** [reply]
DT will analyze the entire portfolio of components every 24 hours at a minimum and will update portfolio, project, and component metrics accordingly.
